pyiceberg/catalog/hive.py

```diff
@@ -271,6 +271,19 @@ def __init__(self, name: str, **properties: str):
             DEFAULT_LOCK_CHECK_RETRIES,
         )
 
+    @staticmethod
+    def _create_hive_client(properties: Dict[str, str]) -> _HiveClient:
+        last_exception = None
+        for uri in properties["uri"].split(","):
+            try:
+                return _HiveClient(uri, properties.get("ugi"))
+            except BaseException as e:
+                last_exception = e
+        if last_exception not None:
```

Comment:
@awdavidson looks like `make lint` failed because of this
```suggestion
        if last_exception is not None:
```